app/mixins/popup.js

```diff
@@ -71,16 +71,15 @@ export default Ember.Mixin.create({
       content: get(this, 'content')
     });
     var self = this;
-    this._popupView._insertElementLater(function() {
-      self._popupView.$().appendTo(self._popup._contentNode);
-      self._popup.update();
-    });
+    this._popupView.createElement();
+    this._popupView.$().appendTo(self._popup._contentNode);
+    this._popup.update();
```

Comment:
This is a delicate part.
Simple tests are passing, but I would like to know if there is a better way to do this.